templates/training/index.html

```diff
@@ -22,7 +22,7 @@ <h1>Train with Ubuntu</h1>
         <div class="col-7">
           <div>
             <p>
-              Train your sysadmins and devops engineers to become <a href="/openstack">Charmed OpenStack</a> or Charmed Kubernetes experts. Canonical runs training programmes to suit all environments and all levels of experience. Explore, prototype and design with cloud technologies so you can get the most out of your environment. All classes and materials are provided in English.
+              Train your sysadmins and devops engineers to become experts in developing and operating open source software. Canonical runs training programmes to suit all environments and all levels of experience. Explore, prototype and design with cloud technologies so you can get the most out of your environment. All classes and materials are provided in English. They can be performed remotely or your premises at extra cost.
```

Comment:
"They can be performed remotely or your premises at extra cost"

Missing a word? "They can be performed remotely or ON your premises at extra cost"